tests/integration/test_end_to_end.py

```diff
@@ -0,0 +1,113 @@
+import pytest
+from mock import patch, Mock
+import time
+
+from gonzo.clouds.compute import Openstack
+from gonzo.scripts import list_, launch
+from gonzo.scripts.base import get_parser
+
+
+""" End to end integration test for spinning up and inspecting live instance.
```

Comment:
this normally goes at the top of the file (before imports)

though i was hoping for a bit more than "devstack recommended". Could we provide enough instructions to get this set up?